tests/test_rect_track.py

```diff
@@ -0,0 +1,104 @@
+# Sanity test for the tracker, tracking small white rectangle.
+import pytest
+import cv2
+import numpy as np
+import sys
+from pathlib import Path
+
+PYTRACKING_PATH = str(Path(__file__).absolute().parent.parent)
+
+sys.path.append(PYTRACKING_PATH)
+
+EVALUATOR_PATH = PYTRACKING_PATH + '/beu-evaluator/'
+CONFIG_PATH = PYTRACKING_PATH + '/pytracking_config.yaml'
+sys.path.append(EVALUATOR_PATH)
+from beu_evaluator import evaluator
+from beu_evaluator.algorithms.pytracking import ObjectTrackerAlgo
+from beu_evaluator.utils import metrics, utils
+
+
+
+TMP_VID_PATH = '/tmp/test_vid.mp4'
+TMP_GT_PATH = '/tmp/test_gt.gt'
```

Comment:
Separate the imports from the globals and consts and re-order things according to PEP8.